committee-steering/governance/project-naming.md

```diff
@@ -0,0 +1,11 @@
+# Project Naming
+
+The Kubernetes project has historically named most of our components/projects with a "kube" substring like `kube-proxy`, `kubectl`, `kube-apiserver`, `kube-controller-manager`, `kube-scheduler`, `minikube`, ...
```

Comment:
Nit: can you line wrap?

Reply:
🤷‍♂️ , this is the sort of thing we should automate with a fixed length if we want it, IMHO.
I left this as one line/list for easier editing.

Reply:
leaving this TODO to come back and fixup later, pending any other fixes (I will squash/fixup the others and update this)